lang/en/docs/materials-designer/header-menu/advanced/jupyterlite-dialog.md

```diff
@@ -0,0 +1,70 @@
+
+At the top of the dialog, a drop-down menu allows for the selection of materials to be transferred to the JupyterLite environment. These materials will then be available for further processing within JupyterLite.
+
+<img src="/images/materials-designer/jupyterlite-transformation-input-materials.png" alt="Select Input Materials"/>
+
+## Apply Transformation
+
+To apply a transformation, open the notebook containing the desired transformation from the list provided in the Introduction.ipynb notebook. Follow the instructions within to apply the transformation to the selected materials. Typically, this process involves specifying settings for the transformation and clicking "Run All Cells."
+
+<img src="/images/materials-designer/jupyterlite-transformation-apply-transformation.png" alt="Apply Transformation"/>
+
+### Access Materials in JupyterLite
+To access materials inside the JupyterLite environment, use function from `utils.jupyterlite` module. The following code snippet demonstrates how to access the materials inside the JupyterLite environment:
+
+```python
+from utils.jupyterlite import get_data
+
+get_data("materials_in", globals())
+```
+
+Parameters:
+
+The first parameter specifies the name of the global variable (`"materials_in"`) where the received data will be stored.
+The second parameter, `globals()`, ensures that the function operates correctly across both Pyodide and Python environments. It allows `get_data` to dynamically interact with the global namespace of the script.
+
+Data Handling:
+
+The materials data is initially stored in a global variable named `data_from_host`, which is updated in response to changes in material selection or the materials themselves.
+In the context of the Pyodide environment, `data_from_host` becomes available after the Pyodide kernel has loaded and the extension set the data.
+
+### Send Materials Back to Materials Designer
+
+To send the materials back to the Materials Designer, use the following code snippet:
+
+```python
+from utils.jupyterlite import send_data
+
+materials = [material1, material2, ...]
+send_data("materials", materials)
+```
+
+Parameters:
+
+The first parameter specifies the data that is being sent, which is "materials" in case for materials, this shouldn't be changed. The second parameter is the list of materials in ESSE format.
+
+
+## Submit Results
+
+In most cases the result of the transformation is a set of materials that can be passed back to the Materials Designer. They will appear in the "Materials Out" dropdown at the bottom of the dialog. Select the materials you want to pass back to the Materials Designer and click the "Submit" button to complete the transformation.
+
+<img src="/images/materials-designer/jupyterlite-transformation-submit-results.png" alt="Submit Results"/>
+
+## Animation
+
+The following animation illustrates how to use the JupyterLite Transformation Dialog to create a matching interface between two surfaces using Zur and McGill's algorithm.
+
```

Comment:
Let's make sure the images are:

- added via LFS
- in webp format
- add explanation to README about webp

Reply:
Last time, we did add brief info about webp in the [INTERNAL.md](https://github.com/Exabyte-io/documentation/blob/master/INTERNAL.md#converting-images-to-webp-format)

Reply:
Probably it's better to have it in the README under "including images" section.

Reply:
Let's keep it in README, probably more visible